packages/actor-rdf-resolve-quad-pattern-federated/lib/FederatedQuadSource.ts

```diff
@@ -12,6 +12,12 @@ import { mapTerms } from 'rdf-terms';
 import type { Algebra } from 'sparqlalgebrajs';
 import { Factory } from 'sparqlalgebrajs';
 
+import type {
+  IActionRdfMetadataAggregate,
+  IActorRdfMetadataAggregateOutput
+} from '@comunica/bus-rdf-metadata-aggregate';
+import {Record} from "immutable";
```

Comment:
This seems incorrect. We should be using `Record` from Node, so just removing this line is fine.